tests/clickhouse-test

```diff
@@ -946,7 +953,9 @@ class TestCase:
             )
 
         if "Exception" in stdout:
-            description += "\n{}\n".format("\n".join(stdout.splitlines()[:100]))
+            description += "\n"
```

Comment:
Consider moving new line prefixes and suffixes inside of `trim_for_log` function